connectors/webhook/element-templates/webhook-connector-boundary.json

```diff
@@ -199,14 +199,14 @@
     },
     "type" : "Dropdown",
     "choices" : [ {
-      "name" : "None",
-      "value" : "NONE"
-    }, {
```

Comment:
Looks like you have an outdated version of the template generator plugin installed locally (template diff is not related to the changes in the PR). Please run `mvn clean install` for the whole project locally and then regenerate the templates again

Reply:
Yeah, the dropdown ordering has been changed a while ago and the current template before your diff seems correct (NONE should be the first, as declared in the annotation). I still suspect that somehow you're using an outdated plugin. Could you try purging the related directory in your local Maven repository? `~/m2/repository/io/camunda/connector`